_sass/module/_candidate-summary.scss

```diff
@@ -13,27 +13,46 @@
   @include grid-column(10);
```

Comment:
Since you're removing one column from the left side, you probably want to add one to the right side (so the two columns take up the full 12-column width). 

But, the image looks pretty small on some screen sizes with a 1-column width. Maybe you want to shrink the image and not the column, or maybe avoid the grid altogether?

800px
![screenshot from 2018-07-25 19-41-12](https://user-images.githubusercontent.com/509703/43238345-bd18b3be-9042-11e8-8772-1e5948ba2101.png)

384px
<img alt="screenshot from 2018-07-25 19-41-50" src="https://user-images.githubusercontent.com/509703/43238372-d1c11fea-9042-11e8-9fc0-1bcba690563a.png" width="300" />